diff --git a/.gitignore b/.gitignore
index c24a835..fa38255 100644
--- a/.gitignore
+++ b/.gitignore
@@ -43,4 +43,6 @@ yarn-error.log*
 *.tsbuildinfo
 
 # idea files
-.idea
\ No newline at end of file
+.idea
+
+bun.lockb
\ No newline at end of file
diff --git a/src/app/editor/_components/actionBar.tsx b/src/app/editor/_components/actionBar.tsx
index 0cd5eeb..ce27407 100644
--- a/src/app/editor/_components/actionBar.tsx
+++ b/src/app/editor/_components/actionBar.tsx
@@ -32,9 +32,9 @@ export function ActionBar({ stageScale, setStageScale }: ActionBarProps) {
     }
     if (value !== "") {
       const newValue = Math.min(Math.max(parseInt(value), 10), 999);
-      setStageScale(newValue / 100);    
+      setStageScale(newValue / 100);
     } else {
-      setStageScaleInput(`${Math.round(stageScale * 100)}%`)
+      setStageScaleInput(`${Math.round(stageScale * 100)}%`);
     }
   };
 
@@ -53,8 +53,14 @@ export function ActionBar({ stageScale, setStageScale }: ActionBarProps) {
   };
 
   return (
-    <div className="flex items-center justify-between rounded-lg bg-[#1F1F1F] p-1.5 pl-2 text-white shadow-[0px_0px_5px_4px_rgba(0,_0,_0,_0.1)]">
-      <Input className="w-min-0 h-7 w-16 bg-[#282828] text-center" value={stageScaleInput} onBlur={handleOnBlur} onChange={handleOnChange} onKeyDown={handleOnKeyDown}/>
+    <div className="sidepanel flex items-center justify-between !p-1.5 !pl-2">
+      <Input
+        className="w-min-0 h-7 w-16 !rounded-sm border border-neutral-700/50 bg-neutral-800 text-center text-neutral-400 hover:text-neutral-100"
+        value={stageScaleInput}
+        onBlur={handleOnBlur}
+        onChange={handleOnChange}
+        onKeyDown={handleOnKeyDown}
+      />
       <div className="flex gap-1.5">
         <TooltipProvider>
           <Tooltip>
@@ -62,8 +68,8 @@ export function ActionBar({ stageScale, setStageScale }: ActionBarProps) {
               <span>
                 <Button
                   size="icon"
-                  className="h-8 w-8 bg-[#5952BB] text-white"
-                  disabled
+                  variant="ghost"
+                  className="h-8 w-8 text-white"
                 >
                   <Download />
                 </Button>
@@ -78,8 +84,8 @@ export function ActionBar({ stageScale, setStageScale }: ActionBarProps) {
               <span>
                 <Button
                   size="icon"
-                  className="h-8 w-8 bg-[#528FBB] text-white"
-                  disabled
+                  variant="ghost"
+                  className="h-8 w-8 text-white"
                 >
                   <Save />
                 </Button>
@@ -94,8 +100,8 @@ export function ActionBar({ stageScale, setStageScale }: ActionBarProps) {
               <span>
                 <Button
                   size="icon"
-                  className="h-8 w-8 bg-[#52BB86] text-white"
-                  disabled
+                  variant="ghost"
+                  className="h-8 w-8 text-white"
                 >
                   <Send />
                 </Button>
diff --git a/src/app/editor/_components/layersPanel.tsx b/src/app/editor/_components/layersPanel.tsx
index 402e927..0b12555 100644
--- a/src/app/editor/_components/layersPanel.tsx
+++ b/src/app/editor/_components/layersPanel.tsx
@@ -1,19 +1,21 @@
 export function LayersPanel() {
   return (
-    <div className="flex-1 rounded-lg bg-[#1F1F1F] px-0.5 pb-2.5 text-white shadow-[0px_0px_5px_4px_rgba(0,_0,_0,_0.1)]">
-      <div className="px-4 pb-2 pt-4 text-xs font-semibold">Elements: non functional</div>
+    <div className="sidepanel flex-1">
+      <div className="px-4 pb-2 pt-4 text-xs font-semibold">
+        Elements: non functional
+      </div>
       <div className="flex flex-col gap-1.5 px-1.5">
-        <div className="rounded-sm px-2.5 py-1 text-xs font-extralight hover:bg-white hover:bg-opacity-5">
-          <span className="font-black pr-2">T</span>Text type
+        <div className="rounded-md bg-white/5 px-2.5 py-1.5 text-xs text-white">
+          <span className="pr-2 font-bold">T</span>Text type
         </div>
-        <div className="rounded-sm px-2.5 py-1 text-xs font-extralight hover:bg-white hover:bg-opacity-5">
-          <span className="font-black pr-2">P</span>Picture type
+        <div className="rounded-md px-2.5 py-1.5 text-xs text-neutral-400 hover:bg-white hover:bg-opacity-5">
+          <span className="pr-2 font-bold">P</span>Picture type
         </div>
-        <div className="rounded-sm px-2.5 py-1 text-xs font-extralight hover:bg-white hover:bg-opacity-5">
-          <span className="font-black pr-2">S</span>Shape type
+        <div className="rounded-md px-2.5 py-1.5 text-xs text-neutral-400 hover:bg-white hover:bg-opacity-5">
+          <span className="pr-2 font-bold">S</span>Shape type
         </div>
-        <div className="rounded-sm px-2.5 py-1 text-xs font-extralight hover:bg-white hover:bg-opacity-5">
-          <span className="font-black pr-2">G</span>Group type
+        <div className="rounded-md px-2.5 py-1.5 text-xs text-neutral-400 hover:bg-white hover:bg-opacity-5">
+          <span className="pr-2 font-bold">G</span>Group type
         </div>
       </div>
     </div>
diff --git a/src/app/editor/_components/propertiesPanel.tsx b/src/app/editor/_components/propertiesPanel.tsx
index 47a2041..d56e48d 100644
--- a/src/app/editor/_components/propertiesPanel.tsx
+++ b/src/app/editor/_components/propertiesPanel.tsx
@@ -6,28 +6,36 @@ interface propertiesPanelProps {
 
 export function PropertiesPanel({ selectedObject }: propertiesPanelProps) {
   return (
-    <div className="flex-1 rounded-lg bg-[#1F1F1F] px-0.5 pb-2.5 text-white shadow-[0px_0px_5px_4px_rgba(0,_0,_0,_0.1)]">
-      <div className="px-3 pb-2 pt-3 text-sm font-bold">
+    <div className="sidepanel flex-1">
+      <div className="px-3 pb-3 pt-3 text-sm font-bold">
         {selectedObject ? selectedObject?.type : "No object is selected"}
       </div>
       {selectedObject && (
         <>
-          <div className="px-3 pb-2 text-xs font-semibold">Position</div>
+          <div className="px-3 pb-1 text-xs font-medium text-neutral-400">
+            Position
+          </div>
           <div className="flex gap-1.5 px-2.5">
-            <div className="w-full rounded-sm bg-white bg-opacity-5 px-2.5 py-1 text-xs font-extralight">
-              <span className="pr-2 font-medium">X</span>{selectedObject.x}
+            <div className="w-full rounded-md bg-white bg-opacity-5 px-2.5 py-1.5 text-xs">
+              <span className="pr-2 font-medium text-neutral-400">X</span>
+              {selectedObject.x}
             </div>
-            <div className="w-full rounded-sm bg-white bg-opacity-5 px-2.5 py-1 text-xs font-extralight">
-              <span className="pr-2 font-medium">Y</span>{selectedObject.y}
+            <div className="w-full rounded-md bg-white bg-opacity-5 px-2.5 py-1.5 text-xs">
+              <span className="pr-2 font-medium text-neutral-400">Y</span>
+              {selectedObject.y}
             </div>
           </div>
-          <div className="px-3 pb-2 pt-3 text-xs font-semibold">Layout</div>
+          <div className="px-3 pb-1 pt-3 text-xs font-medium text-neutral-400">
+            Dimensions
+          </div>
           <div className="flex gap-1.5 px-2.5">
-            <div className="w-full rounded-sm bg-white bg-opacity-5 px-2.5 py-1 text-xs font-extralight">
-              <span className="pr-2 font-medium">W</span>{selectedObject.width}
+            <div className="w-full rounded-md bg-white bg-opacity-5 px-2.5 py-1.5 text-xs">
+              <span className="pr-2 font-medium text-neutral-400">W</span>
+              {selectedObject.width}
             </div>
-            <div className="w-full rounded-sm bg-white bg-opacity-5 px-2.5 py-1 text-xs font-extralight">
-              <span className="pr-2 font-medium">H</span>{selectedObject.height}
+            <div className="w-full rounded-md bg-white bg-opacity-5 px-2.5 py-1.5 text-xs">
+              <span className="pr-2 font-medium text-neutral-400">H</span>
+              {selectedObject.height}
             </div>
           </div>
         </>
diff --git a/src/app/editor/_components/variantsPanel.tsx b/src/app/editor/_components/variantsPanel.tsx
index fcc0bb5..7bbce9f 100644
--- a/src/app/editor/_components/variantsPanel.tsx
+++ b/src/app/editor/_components/variantsPanel.tsx
@@ -13,17 +13,17 @@ export function VariantsPanel({
   setSelectedObject,
 }: VariantsPanelProps) {
   return (
-    <div className="rounded-lg bg-[#1F1F1F] pb-2.5 px-0.5 text-white shadow-[0px_0px_5px_4px_rgba(0,_0,_0,_0.1)]">
-      <div className="items-center px-4 pt-4 pb-2.5">
+    <div className="sidepanel">
+      <div className="items-center px-4 pb-2.5 pt-4">
         <P_svg />
       </div>
-      <div className="px-4 font-extralight pb-1">Untitled</div>
-      <div className="px-4 pb-2 text-xs font-semibold">Variants</div>
+      <div className="px-4 pb-3 font-medium outline-none">Untitled</div>
+      <div className="px-4 pb-1 text-xs text-neutral-400">Variants</div>
       <div className="flex flex-col gap-1.5 px-1.5">
         {frames.map((frame) => (
           <div
             key={frame.id}
-            className={`rounded-sm px-2.5 py-1 text-xs font-extralight hover:bg-white hover:bg-opacity-5 ${selectedObject?.id === frame.id ? "bg-white bg-opacity-5" : ""}`}
+            className={`rounded-md px-2.5 py-1.5 text-xs hover:bg-white hover:bg-opacity-5 ${selectedObject?.id === frame.id ? "bg-white bg-opacity-5" : ""}`}
             onClick={() => setSelectedObject(frame)}
           >
             {frame.name}
diff --git a/src/app/editor/page.tsx b/src/app/editor/page.tsx
index dae2f3f..298619e 100644
--- a/src/app/editor/page.tsx
+++ b/src/app/editor/page.tsx
@@ -99,7 +99,7 @@ export default function EditorPage() {
     }
   };
 
-/*   const handleContextMenuClosing = () => {
+  /*   const handleContextMenuClosing = () => {
     // Dispatch 'Escape' key event to close context menu
     const escapeEvent = new KeyboardEvent("keydown", { key: "Escape" });
     document.dispatchEvent(escapeEvent);
@@ -309,7 +309,10 @@ export default function EditorPage() {
   }
 
   return (
-    <MenuWrapper selectedObject={selectedObject} deleteObject={deleteSelectedObject}>
+    <MenuWrapper
+      selectedObject={selectedObject}
+      deleteObject={deleteSelectedObject}
+    >
       <div className="relative h-full w-full">
         <Stage
           width={dimensions.width}
diff --git a/src/styles/globals.css b/src/styles/globals.css
index 2650250..65f0ef5 100644
--- a/src/styles/globals.css
+++ b/src/styles/globals.css
@@ -27,7 +27,7 @@
     --chart-3: 197 37% 24%;
     --chart-4: 43 74% 66%;
     --chart-5: 27 87% 67%;
-    --radius: 0.5rem
+    --radius: 0.5rem;
   }
   .dark {
     --background: 240 10% 3.9%;
@@ -53,7 +53,7 @@
     --chart-2: 160 60% 45%;
     --chart-3: 30 80% 55%;
     --chart-4: 280 65% 60%;
-    --chart-5: 340 75% 55%
+    --chart-5: 340 75% 55%;
   }
 }
 @layer base {
@@ -72,4 +72,10 @@
   body {
     @apply bg-background text-foreground;
   }
-}
\ No newline at end of file
+}
+
+@layer components {
+  .sidepanel {
+    @apply rounded-lg border border-neutral-800 bg-[#1F1F1FEB] px-0.5 pb-2.5 text-white shadow-[0px_0px_5px_4px_rgba(0,_0,_0,_0.05)] backdrop-blur-lg;
+  }
+}
